README.md

```diff
@@ -1,4 +1,4 @@
-Text progress bar library for Python.
+#Text progress bar library for Python
```

Comment:
Please add a space after #. I think the Markdown parser needs it to interpret it correctly.